### PR TITLE
Clean up temp dir on success

### PR DESCRIPTION
**Pull request by giang-nghg**

### CODE REVIEW

File: ivcfmerge/batch.py [hunk 1]
```diff
@@ -29,8 +30,16 @@ def ivcfmerge_batch(input_paths, output_path, batch_size=None, temp_dir=None):
...
+
 
 def _ivcfmerge(input_paths, output_path):
     with ExitStack() as stack, open(output_path, 'w+') as outfile:
         infiles = map(lambda p: stack.enter_context(open(p)), input_paths)
         ivcfmerge(infiles, outfile)
+
+
+def clean_up_temp_dir(temp_dir):
+    for path in Path(temp_dir).glob('*'):
```
Comment (Zhicheng-Liu) [inline ivcfmerge/batch.py]: I don't think you can remove everything under the temp_dir. What if the temp_dir happens to contain other files?

Comment (giang-nghg) [reply ivcfmerge/batch.py]: @Zhicheng-Liu Fixed! Thanks!

Comment (giang-nghg) [reply ivcfmerge/batch.py]: @Zhicheng-Liu also added a test to ensure this condition: https://github.com/iqbal-lab-org/ivcfmerge/pull/8/files#diff-71f839c3e1c7e69cdcc25d7da2c1860eR81

File: tests/cli/test_ivcfmerge_batch_cli.py [hunk 2]
```diff
@@ -24,11 +27,32 @@ def test_batch_sizes(input_paths_file, ref_merged_path, tmp_path, batch_size):
...
 
     subprocess.run(['python3', 'ivcfmerge_batch.py', '--batch-size', str(batch_size),
-                    '--temp-dir', str(tmp_path), input_paths_file, str(output_path)])
+                    '--temp-dir', str(temp_dir), input_paths_file, str(output_path)])
 
     with output_path.open() as outfile, open(ref_merged_path, 'r') as ref_merged:
         assert outfile.read() == ref_merged.read()
+
+
+@given(batch_size=st.integers(min_value=2, max_value=5))
```
Comment (Zhicheng-Liu) [inline tests/cli/test_ivcfmerge_batch_cli.py]: Can we change the magic number 5 to `len(_input_paths())-1`? Same for other places.

**Pull request merged**